generators/entity-client/templates/react/src/main/webapp/app/entities/entity.reducer.ts.ejs

```diff
@@ -55,6 +55,7 @@ const initialState = {
   entity: {},
   updating: false,
   updateSuccess: false
+  <%_ if (pagination !== 'no') { _%>, totalItems: 0<%_ } _%>
```

Comment:
move this above `updateSuccess` so that you can so this cleanly
```
<%_ if (pagination !== 'no') { _%>
totalItems: 0,
<%_ } _%>
updateSuccess: false
```